examples/todo-list/src/controllers/todo-list-todo.controller.ts

```diff
@@ -13,33 +13,68 @@ export class TodoListTodoController {
     @repository(TodoListRepository) protected todoListRepo: TodoListRepository,
   ) {}
 
-  @post('/todo-lists/{id}/todos')
-  async create(@param.path.number('id') id: number, @requestBody() todo: Todo) {
+  @post('/todo-lists/{id}/todos', {
+    responses: {
+      '200': {
+        description: 'TodoList.Todo model instance',
+        content: {'application/json': {'x-ts-type': Todo}},
```

Comment:
Just in case it was not clear, if we added `modelRef(ctor)` helper as a sugar API returning `{'x-ts-type': ctor}`, then the line above could look like this:

```ts
content: {'application/json': modelRef(Todo)},
```

Having wrote that, I am not sure how much better that is and whether the benefits justify maintenance costs. I guess what we really want is to implement the new decorator.

Reply:
I'd rather see new decorators than introduce a helper utility that we have to maintain.